src/alarmhandler.h

```diff
@@ -0,0 +1,58 @@
+/*
+  This file is part of the mkcal library.
+
+  Copyright (c) 2023 Damien Caliste <dcaliste@free.fr>
+
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Library General Public
+  License as published by the Free Software Foundation; either
+  version 2 of the License, or (at your option) any later version.
+
+  This library is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Library General Public License for more details.
+
+  You should have received a copy of the GNU Library General Public License
+  along with this library; see the file COPYING.LIB.  If not, write to
+  the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+  Boston, MA 02110-1301, USA.
+*/
+/**
+  @file
+  This file is part of the API for handling calendar data and
+  defines the an interface to store a single KCalendarCore::Calendar.
```

Comment:
Some case for "the an" was fixed, but there are still multiple ones copypasting it and the text looks wrong in many of them altogether, like here.

Reply:
Thanks for noticing. I've cleaned up a bit the documentation strings, and in #60 PR also.

I've added documentation strings for the internal `calendarhandler_p.h` and `alarmhandler_p.h` files also.